ivy/functional/backends/tensorflow/elementwise.py

```diff
@@ -600,10 +596,7 @@ def positive(
     return tf.experimental.numpy.positive(x)
 
 
-@with_unsupported_dtypes(
-    {"2.13.0 and below": ("uint8", "uint16", "uint32", "uint64", "float64")},
-    backend_version,
-)
+@with_unsupported_dtypes({"2.13.0 and below": ("bool", "unsigned")}, backend_version)
```

Comment:
have you tested `float64`?

Reply:
yes, I rerun it and `float64` does not cause an issue
![image](https://github.com/unifyai/ivy/assets/100947451/2425b55f-68d0-447a-9867-4a12eff01180)

Reply:
awesome, thanks for clarifying!